metau-capstone/LoginViewController.m

```diff
@@ -0,0 +1,67 @@
+//
+//  LoginViewController.m
+//  metau-capstone
+//
+//  Created by Shirley Zhu on 7/6/22.
+//
+
+#import "LoginViewController.h"
+#import "Parse/Parse.h"
+
+@interface LoginViewController ()
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+- (IBAction)loginUser:(id)sender {
+    NSString *username = self.usernameField.text;
+    NSString *password = self.passwordField.text;
+    
+    [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
+        if (error != nil) {
+            NSLog(@"User log in failed: %@", error.localizedDescription);
+        } else {
+            NSLog(@"User logged in successfully");
+            
+            // display view controller that needs to shown after successful login
+        }
+    }];
+}
+
+- (IBAction)registerUser:(id)sender {
+    // initialize a user object
+    PFUser *newUser = [PFUser user];
+    
+    // set user properties
+    newUser.username = self.usernameField.text;
+    newUser.password = self.passwordField.text;
```

Comment:
things to think about, what about input validation & empty character truncation?
- english character only?
- truncate empty character before and after the input?
- double passwords with confirmation capability
- hide as passwords as user type
- option to view password